generator/internal/stub_rest_generator.cc

```diff
@@ -55,6 +55,9 @@ Status StubRestGenerator::GenerateHeader() {
   std::vector<std::string> additional_pb_header_paths =
       absl::StrSplit(vars("additional_pb_header_paths"), absl::ByChar(','));
   HeaderSystemIncludes(additional_pb_header_paths);
+  std::vector<std::string> mixin_headers =
+      absl::StrSplit(vars("mixin_proto_header_paths"), ',');
```

Comment:
super nit, feel free to ignore: there is something funny about this and L56, which do the same exact thing with slightly different syntax. I don't really care though.